src/components/layout/signIn.jsx

```diff
@@ -21,7 +48,14 @@ const SignIn = ({ switchForm }) => {
         <label className={styles.formName} htmlFor='email'>
```

Comment:
I would also suggest a form here.

Reply:
John worked on all the design aspects. I just added the API and made some changes accordingly. I’ll check with John about this and make any necessary updates. Thank you